src/rbm10_VIC.f90

```diff
@@ -17,113 +17,110 @@ PROGRAM RBM10_VIC
 character (len=200 ):: heat_file
-character (len=200 ):: net_file
-character (len=200 ):: param_file
-character (len=200 ):: temp_file
+character (len=200 ):: net_file
+character (len=200 ):: temp_file
 character (len=200 ):: spatial_file
-character (len=8)   :: start_data,end_data     
-integer iargc
-integer numarg
-
-!
-! Command line input
-!
-numarg = iargc ( )
-if (numarg .lt. 2) then
-  write (*,*) 'Too few arguments were given'
-  write (*,*) ' '
-  write (*,*) 'First:  Location and prefix of input files'
-  write (*,*) '        (networkfile and parameterfile)'
-  write (*,*) 'Second: Location and prefix of output files'
-  write (*,*) ' '
-  write (*,*) 'eg: $ <program-name> ./input/Salmon_0.50 ./output/Salmon_Test'
-  write (*,*) ' '
-  stop
-end if
-call getarg ( 1, inPrefix )
-call getarg ( 2, outPrefix )
-!
+character (len=8)   :: start_data,end_data     
+integer iargc
+integer numarg
+
+!
+! Command line input
+!
+numarg = iargc ( )
+if (numarg .lt. 2) then
+  write (*,*) 'Too few arguments were given'
+  write (*,*) ' '
+  write (*,*) 'First:  Location and prefix of input files'
+  write (*,*) '        (networkfile and parameterfile)'
+  write (*,*) 'Second: Location and prefix of output files'
+  write (*,*) ' '
+  write (*,*) 'eg: $ <program-name> ./input/Salmon_0.50 ./output/Salmon_Test'
+  write (*,*) ' '
+  stop
+end if
+call getarg ( 1, inPrefix )
+call getarg ( 2, outPrefix )
+!
 ! Identify input/output files
-!
-net_file      = TRIM(inPrefix)//'_Network'
-param_file    = TRIM(inPrefix)//'_Parameters'
-spatial_file  = TRIM(outPrefix)//'.Spat'
-temp_file     = TRIM(outPrefix)//'.Temp'
-!
-write(*,*) 'Spatial file: ',spatial_file         
-write(*,*) 'Network file    : ',net_file
-write(*,*) 'Parameter file  : ',param_file!
-write(*,*) 'Temperature file: ',temp_file
-!
-OPEN(UNIT=90,FILE=TRIM(net_file),STATUS='OLD')
-!
-!     Read header information from control file
-!
-read(90,*)
-read(90,'(A)') flow_file
-!
-!     Open file with hydrologic data
-!
-open(unit=35,FILE=TRIM(flow_file) ,FORM='FORMATTED',ACCESS='DIRECT' ,RECL=60,STATUS='old')
-!
-!
-read(90,'(A)') heat_file
-!
-!     Open file with meteorologic data
-!     
-open(unit=36,FILE=TRIM(heat_file) ,FORM='FORMATTED',ACCESS='DIRECT' ,RECL=50,STATUS='old')
-!
-!     Call systems programs to get started
-!
-!     SUBROUTINE BEGIN reads control file, sets up topology and
-!     important properties of reaches
-!
+!
+net_file      = TRIM(inPrefix)//'_Network'
+spatial_file  = TRIM(outPrefix)//'.Spat'
+temp_file     = TRIM(outPrefix)//'.Temp'
+!
+write(*,*) 'Spatial file: ',spatial_file         
+write(*,*) 'Network file    : ',net_file
+write(*,*) 'Temperature file: ',temp_file
+!
+OPEN(UNIT=90,FILE=TRIM(net_file),STATUS='OLD')
+!
+!     Read header information from control file
+!
+read(90,*)
+read(90,'(A)') flow_file
+!
+!     Open file with hydrologic data
+!
+open(unit=35,FILE=TRIM(flow_file), ACCESS='SEQUENTIAL',FORM='FORMATTED', STATUS='old')
```

Comment:
I changed this line in order to read the new format of flow and energy data; same as line 101 below